_posts/2015-07-06-code-reviews-for-learning.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "Why Code Reviews Are Great Learning Tools"
+description: "Code reviews are a good way to make sure (or at least, somewhat more sure) that crapp or buggy code does not make its way to production. But, code reviews are also a terrific way to consolidate your learning. Here's why!"
+tags: ["learning", "resources", "hackership"]
+authors:
+ - anouk
+image: /content/2015-07-06-code-review-whiteboard.jpg
+---
+
```

Comment:
Without the introduction paragraph from the top, this article doesn't make clear that it focusses on the _learning_ aspects of code reviews. It'd really stress that because there are plenty other benefits, we aren't discussion (like quality assurance, shared ownership, etc).